lambdas/handlers/back_channel_logout_handler.py

```diff
@@ -22,8 +24,16 @@
 def lambda_handler(event, context):
     request_context.app_interaction = LoggingAppInteraction.LOGOUT.value
     logger.info("Back channel logout handler triggered")
-
-    back_channel_logout_service = BackChannelLogoutService()
+    feature_flag_service = FeatureFlagService()
+    auth_flag_name = FeatureFlags.USE_SMARTCARD_AUTH.value
+    use_smartcard_lambda_enabled_flag_object = (
+        feature_flag_service.get_feature_flags_by_flag(auth_flag_name)
+    )
+    if use_smartcard_lambda_enabled_flag_object[auth_flag_name]:
+        auth_ssm_prefix = "/auth/smartcard/"
+    else:
+        auth_ssm_prefix = "/auth/password/"
+    back_channel_logout_service = BackChannelLogoutService(auth_ssm_prefix)
```

Comment:
Would it be possible to just initiate the FeatureFlagService() at oidc_service and load the flag there?  It seems a bit odd that we have to load a hard-coded string at the top level, and then pass that all along to a bottom-level

Reply:
I was thinking about it too, but it seems odd to have a configuration nested in the oidc service.

Reply:
all right, thanks!